jobs/service-fabrik-broker/spec

```diff
@@ -181,6 +189,15 @@ properties:
   internal.domain_socket.path:
     description: "domain socket path used to communicate with broker monitoring agent"
     default: /tmp/sfevents
+  internal.floating_ip:
+    description: "Floating IP used for Ip re-assignment in HA setup"
+    default: 10.11.252.10
+  internal.master_ip:
```

Comment:
Will you keep both internal.ip as well as internal. floating_ip ? Why not only one of them?

Reply:
+1